src/content/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements.mdx

```diff
@@ -53,29 +49,29 @@ Make sure your Android app meets these requirements:
       </td>
 
       <td>
-        HttpURLConnection, OkHttp2 (except versions 2.0 and 2.4), OkHttp (versions 2.8, 3.5+, 4.0+), OkIO (version 1.11), AndroidHttpClient, Volley (version 1.0.0), and Apache HTTP Client networking APIs
+HttpURLConnection, OkHttp2 (except versions 2.0 and 2.4), OkHttp (versions 2.8, 3.5+, 4.0+), OkIO (version 1.11), AndroidHttpClient, Volley (version 1.0.0), and Apache HTTP Client networking APIs
 
-        <Callout variant="important">
-        `AndroidHttpClient` was removed from the SDK in version 23, and as such only supported in agent version `6.5.0` and below
-        </Callout>
+Android agent version 6.6.0 or higher requires Android SDK Tools version 24 or higher. For Android SDK Tools version 24 or higher, you also need to meet the following requirements:
 
-        Build with Android SDK Tools version 24 or higher and the following as appropriate:
+  * Android Studio
+  * Gradle 5.1.1
+  * Gradle plugin 3.4
+  * Proguard 5.0
+  * DexGuard 8.0 through 9.x
 
-        * Android Studio
-        * Gradle 5.1.1
-        * Gradle plugin 3.4
-        * Proguard 5.0
-        * DexGuard 8.0 through 9.x
+For older Android agent versions, your build's Android SDK Tools version must be compatible with your Android agent version:
+  * Android agent version 6.0.0 through 6.5.0 requires builds to be compiled with*Android SDK Tools version 21 or higher.
+  * Android agent version 5.0.0 through 5.9.9 requires builds to be compiled with Android SDK Tools version 14 or higher.
 
-          <Callout variant="important">
-            As of January 2021, our Android agent discontinued support for Android Gradle Plugin version 2. For more information, see our [Support Forum post](https://discuss.newrelic.com/t/important-upcoming-changes-to-capabilities-across-browser-php-python-mobile-and-android-agents/123951).
-          </Callout>
+Note: Make sure to upgrade your SDK Tools to the most recent version before you install the Android agent.
 
-          <Callout variant="important">
-            As of May, 2022, Android Gradle plugin version 7.2.1 release breaks the transform API when used along with ASM API, and this has impacted New Relic Android agent functionalities. This issue has been fixed in [Android Gradle plugin version 7.2.2](https://developer.android.com/studio/releases/gradle-plugin#7-2-0)
-          </Callout>
+<Callout variant="important">
+  `AndroidHttpClient` was removed from the SDK in version 23, and as such only supported in agent version `6.5.0` and below
```

Comment:
2 comments here:
- AndroidHttpClient is written above but without the ``, so I think that both should be written the same way, with `` or without ``.
- Per our [style guide](https://docs.newrelic.com/docs/style-guide/word-choice/usage-dictionary/#version), I'd replace `below` with `lower` and add the final full stop.